### ISSUE TITLE
[Feature]: Base64 files larger than 5 Mb uploaded to a Postgres database fail to display 

### ISSUE THREAD

**Issue by felix-appsmith** [appsmithorg/appsmith#17635]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Currently, it is not possible to easily upload a PDF file larger than 5 MB to a Postgres database and display it in an Iframe or document viewer because files larger than 5mb are converted to blob and blob accesses a URL to which widgets cannot access directly

### Why should this be worked on?

It would save users headaches by not having to write their JS code so that they can handle the blob files generated by Appsmith.
This issue is an example of the problems that can cause this to be unavailable: [issue](https://discord.com/channels/725602949748752515/1030154111775150160)

**Comment by Nikhil-Nandagopal** [comment]
So the problem is not with inserting, it's with displaying the file which is > 5mb? @felix-appsmith would it work if the Document viewer widget simply did this out of the box?

**Comment by dilippitchika** [comment]
I didn't get this clearly, is the problem that document viewer cannot work with large base64 data?

**Comment by Nikhil-Nandagopal** [comment]
@dilippitchika yes because larger files encode their base64 as a blob in Appsmith. So if you bind a filepicker file that is larger than 5mb, it will show as blob://<path to file>
This is substituted on the fly for queries but I think for other widgets like document viewer, it is not

**Comment by dilippitchika** [comment]
Cool, this looks like we can add specific logic for this to work on the document viewer. Will pick this up after scoring the issue.
